### ISSUE TITLE
On iOS 13, allowEditor param does not work 

### ISSUE THREAD

**Issue by jamieomatthews** [EurekaCommunity/ImageRow#73]
- Environment: ImageRow, Xcode 10.1, 10.3, and 11.2 (tried all, just in case).  
- On iOS 13 - allowEditor does not work, and thus, when the image is picked, nothing is returned because we search for the `UIImagePickerController.InfoKey.editedImage` which is nil. 

Most likely - this is due to a difference in how iOS 13+ works with UIImagePickerController.  We set the allowsediting flag on the image picker in viewDidLoad of the subclass, which is likely too late.  We can fix it by setting this value when we init the controller instead.

```
 presentationMode = .presentModally(controllerProvider: ControllerProvider.callback {
        let controller = ImagePickerController()
        controller.allowsEditing = self.allowEditor
        return controller
        }, onDismiss: { [weak self] vc in
        self?.select()
      vc.dismiss(animated: true)
    })
```

**Comment by mats-claassen** [comment]
Thanks for reporting this. We should update the Readme with this

**Comment by jamieomatthews** [comment]
@mats-claassen what do you mean update the readme?  The change has to be made in the library itself (unless I'm misunderstanding).  If you were ok with this fix - I can created a PR.

**Comment by mats-claassen** [comment]
Sorry, you are right. Please create a PR with that change

**Comment by jamieomatthews** [comment]
Fixed by #74
